design/ceph/ceph-cluster-cleanup.md

```diff
@@ -7,8 +7,11 @@ As a rook user, I want to clean up data on the hosts after I intentionally unins
 ## Background
 
 ### Host-based cluster deletion
+If the user deletes a host-based cluster (running on raw devices and not on PVs) and starts a new cluster on the same hosts, then the user needs to perform the following manual steps:
```

Comment:
The "not on PVs" mention is confusing, you could make it use of local drives with PVs and have them being sanitized by the current implementation.

Reply:
Update the text. Keeping it more generic now.